presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -50,13 +52,23 @@ public final void destroy()
     }
 
     @Test
-    public void testDropTable()
+    public void testTableLifecycleWithNamesContainingUnderscores()
             throws Exception
     {
-        assertQueryTrue("CREATE TABLE test_drop AS SELECT 123 x");
-        assertTrue(queryRunner.tableExists(getSession(), "test_drop"));
+        Session session = testSessionBuilder()
+                .setCatalog("mysql")
+                .setSchema("test_database")
+                .build();
 
-        assertQueryTrue("DROP TABLE test_drop");
-        assertFalse(queryRunner.tableExists(getSession(), "test_drop"));
+        String tableName = "test_table_with_name_containing_underscores";
+        assertFalse(queryRunner.tableExists(session, tableName));
+
+        assertQuery(session, "CREATE TABLE " + tableName + " AS SELECT 123 x", "SELECT true");
```

Comment:
It is preferable to use `assertQueryTrue` for these. We should add a version that takes session.